apps/ejabberd/src/mod_privacy.erl

```diff
@@ -154,25 +154,26 @@ stop(Host) ->
 %% Handlers
 %% ------------------------------------------------------------------
 
-process_iq_get(_,
-        _From = #jid{luser = LUser, lserver = LServer},
-        _To,
-        #iq{xmlns = ?NS_PRIVACY, sub_el = #xmlel{children = Els}},
-        #userlist{name = Active}) ->
-    case xml:remove_cdata(Els) of
-        [] ->
-            process_lists_get(LUser, LServer, Active);
-        [#xmlel{name = Name, attrs = Attrs}] ->
-            case Name of
-                <<"list">> ->
-                    ListName = xml:get_attr(<<"name">>, Attrs),
-                    process_list_get(LUser, LServer, ListName);
-                _ ->
-                    {error, ?ERR_BAD_REQUEST}
-            end;
-        _ ->
-            {error, ?ERR_BAD_REQUEST}
-    end;
+process_iq_get(Acc,
+               _From = #jid{luser = LUser, lserver = LServer},
```

Comment:
All right, could you please note the ideas for additional changes? I agree this can be done in a separate PR(s).